internal/bminventory/inventory.go

```diff
@@ -6465,7 +6446,8 @@ func (b *bareMetalInventory) V2UpdateHostIgnitionInternal(ctx context.Context, p
 		return nil, common.NewApiError(http.StatusInternalServerError, err)
 	}
 
-	b.eventsHandler.AddEvent(ctx, params.InfraEnvID, &params.HostID, models.EventSeverityInfo, fmt.Sprintf("Host %s: custom discovery ignition config was applied", hostutil.GetHostnameForMsg(&h.Host)), time.Now())
+	eventgen.SendHostDiscoveryIgnitionConfigAppliedEvent(ctx, b.eventsHandler, &params.InfraEnvID, params.HostID, params.InfraEnvID,
```

Comment:
the third argument should be cluster-id

Reply:
removed `cluster-id` from the event's properties